buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -83,12 +83,13 @@ object Libraries {
         const val desugaring = "1.1.5"
         const val workManager = "2.8.1"
         const val fragment = "1.5.6"
-        const val compose = "1.4.1"
+        const val compose = "1.5.0-beta03"
```

Comment:
Are we releasing a new hotfix?

`dependencies.kt` looks very different on `develop` now.

Reply:
Yes, this PR is considered a fix for all remaining NavBackStackEntry crashes, I just wanted to upgrade these libraries that are necessary to get rid of these crashes. @vitorhugods do you think we shouldn't update RC with these changes at this point?